### ISSUE TITLE
Stop malformed CGI output from aborting server

### ISSUE THREAD

**Issue by monsanto** [smogon/infra#11]
PHP code like `header("Content-Type")` will cause infra to abort with an error like:

```
/home/monsanto/smogonrepos/infra/node_modules/header-stack/parser.js:109
      return this.emit('error', new Error('ParseError: Malformed header line, no delimiter (:) found: "' + line + '"'));
```

`cgi` doesn't seem to have a way of catching this error, probably just need to vendor the code and make the tweak manually, it seems to be abandonware anyway

**Comment by Zarel** [comment]
`header("Content-Type")` shouldn't be called ever; is it being called in a dep?

**Comment by monsanto** [comment]
Nah, I was stress testing some stuff. I agree that this shouldn't ever happen
